services/Mumble/MumblePost.ts

```diff
@@ -105,13 +117,64 @@ export class MumblePostService extends MumbleService {
     };
   }
 
+  public async updatePostText({
+    token,
+    text,
+    postIdentifier,
+  }: IUpdatePost): Promise<IUpdatePostTextMumble> {
+    try {
+      // response is empty
+      await this.performRequest({
+        method: EApiMethods.PATCH,
+        path: `${EEndpointsBackend.POSTS}/${postIdentifier}`,
+        token,
+        message: 'Updating post',
+        expectedBack: 'text', // do we need this?
```

Comment:
have you tried without it?

Reply:
yes. it failed. It depends on the headers (accept) you sending. Probably this is a part to improve but generally. not only here.